src/main/webapp/js/datepicker.js

```diff
@@ -89,7 +93,8 @@ function getMaxDateForVisibleDate(startDate, publishDate) {
 
 /**
  * @assumption: visibleDate has a valid value
- * @returns {Date} visibleDate
+ * @param {Date} visibleDate
+ * @returns {Date}
  */
 function getMinDateForPublishDate(visibleDate) {
     return visibleDate;
```

Comment:
@xpdavid i have a doubt regarding both these functions
`$('#visibledate').datepicker({
        dateFormat: 'dd/mm/yy',
        showOtherMonths: true,
        gotoCurrent: true,
        defaultDate: yesterday,
        maxDate: today,
        onSelect: function() {
            var newPublishDate = getMinDateForPublishDate($('#visibledate').datepicker('getDate'));
            $('#publishdate').datepicker('option', 'minDate', newPublishDate);
        }
    });

    $('#publishdate').datepicker({
        dateFormat: 'dd/mm/yy',
        showOtherMonths: true,
        gotoCurrent: true,
        defaultDate: tomorrow,
        onSelect: function() {
            var newVisibleDate = getMaxDateForVisibleDate($('#startdate').datepicker('getDate'),
                    $('#publishdate').datepicker('getDate'));
            $('#visibledate').datepicker('option', 'maxDate', newVisibleDate);
        }
    });`

Here the function is used to set minDate option of ('#publishDate').datepicker() but there is no minDate option.
Also why publishDate has relation with visibleDate?,i think it should be related with startDate as in this case:
(dd/mm/yyyy)
visibleDate :08/03/2017
StartDate:09/03/2017
minDate option for publishDat0e will set to 08/03/2014
but we cannot publish session results before the session has even started?